Test notebooks github actions (#145)
atoti/notebooks: CONTRIBUTING.md

- python methods should be typed
- documentation should follow Google's [docstring](http://google.github.io/styleguide/pyguide.html#381-docstrings)

### Testing and CI

> This repository is tested using [github actions](.github/workflows/test.yaml).

Reviewer: ```suggestion
This repository is tested with a [Github Action](.github/workflows/test.yaml).
```

> - Reformat notebooks using black :

Reviewer: ```suggestion
These commands can be used while developing: 

- Reformat notebooks using black:
```

>   `poetry run black-nb notebooks/`

Reviewer: ```suggestion
```bash
poetry run black-nb notebooks/
```
```

> - Check notebook execution:

Reviewer: ```suggestion
- Execute all the notebooks:
```

>   `poetry run python .github/workflows/execute_notebooks.py`

Reviewer: Use ```bash``` here too. Seeing the path to the Python file here, it would be nicer to move it to a place like `tests/execute_notebooks.py` rather than inside `.github/workflows`.

> Add notebooks creating data or preprocessing data to the [DATA_PREPROCESSING_NOTEBOOKS list](.github/workflows/execute_notebooks.py) if the execution of these notebooks take too much time or need specific credentials.

Reviewer: ```suggestion
```

> Checkout [poetry](https://python-poetry.org/docs/) for more info.

Reviewer: Do you have something specific in mind here? If no, I would rather remove this line.
